cluster/terraform_kubernetes/config/prometheus/alertmanager/app_alert.rules.tmpl

```diff
@@ -34,7 +34,7 @@ groups:
   rules:
 %{ for instance in apps ~}
   - alert: High Memory Utilisation for ${instance.app_name}
-    expr: sum by (namespace, container) (container_memory_working_set_bytes{container!="",image!="",namespace="development",pod=~"apply-review-nnnn-[a-z0-9]+-[a-z0-9]+"})/sum by (namespace, container) (kube_pod_container_resource_requests{container!="",namespace="development",pod=~"apply-review-nnnn-[a-z0-9]+-[a-z0-9]+",resource="memory"}) > ${instance.max_mem}
+    expr: sum by (namespace, container) (container_memory_working_set_bytes{container!="",image!="",namespace="${instance.namespace}",pod=~"${instance.app_name}-[a-z0-9]+-[a-z0-9]+"})/sum by (namespace, container) (kube_pod_container_resource_requests{container!="",namespace="${instance.namespace}",pod=~"${instance.app_name}-[a-z0-9]+-[a-z0-9]+",resource="memory"}) > ${instance.max_mem}
```

Comment:
while in most cases we have memory requests = memory limits, shouldn't this use kube_pod_container_resource_limits as that's not guaranteed

Reply:
changes to kube_pod_container_resource_limits